templates/evcc.io/brands.json

```diff
@@ -134,7 +134,7 @@
   "Senergy",
   "Shelly",
   "SMA",
-  "SOFASOLAR",
+  "SOFARSOLAR",
```

Comment:
Die Firma SofarSolar schreibt sich selbst in Camel Case auf ihren Seiten.

Reply:
Den Kommentar hatte ich auch. Tatsächlich haben sie 3 Varianten auf ihrer Seite… Sollen wir uns auf CamelCase einigen?

Reply:
Nur meine persönliche Meinung: Ich fände es unfair den anderen Herstellern gegenüber SofarSolar durch ALLCAPS hervorzuheben. Bei echten Abkürzungen wie SMA ist das natürlich was anderes.

Reply:
So soll es sein!